### PR TITLE
libtirpc 1.3.4

### PR DESCRIPTION
**Pull request by woodruffw**
There are some inexplicable linkage errors
happening when rebottling. Maybe bumping will fix
it.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by woodruffw** [comment]
This is what I get for not searching closed PRs: https://github.com/Homebrew/homebrew-core/pull/150136

No response from the upstream here: <https://sourceforge.net/p/libtirpc/mailman/message/51781335/>

**Comment by woodruffw** [comment]
I was hoping that `--enable-authdes` would inject a soft implementation of `des_setparity`, but no such luck. So this is still hard blocked on upstream, and we should probably consider removing it outright.

**Comment by chenrui333** [comment]
gonna reuse this PR a bit.

File: Formula/lib/libtirpc.rb [hunk 1]
```diff
@@ -13,23 +13,20 @@ class Libtirpc < Formula
...
-      "--disable-dependency-tracking",
-      "--disable-silent-rules",
-      "--prefix=#{prefix}"
+    system "./configure", "--disable-silent-rules", *std_configure_args.reject { |s| s["--disable-debug"] }
     system "make", "install"
   end
 
   test do
     (testpath/"test.c").write <<~EOS
-      #include <rpc/des_crypt.h>
```
Comment (chenrui333) [inline Formula/lib/libtirpc.rb:25]: updated the test to not test `des_crypt`

File: Formula/lib/libtirpc.rb [hunk 2]
```diff
@@ -13,23 +13,20 @@ class Libtirpc < Formula
...
+      #include <rpc/rpc.h>
       #include <stdio.h>
-      int main () {
-        char key[] = "My8digitkey1234";
-        if (sizeof(key) != 16)
-          return 1;
-        des_setparity(key);
-        printf("%lu\\n", sizeof(key));
+
+      int main() {
```
Comment (woodruffw) [inline Formula/lib/libtirpc.rb:25]: I might be missing something, but I think this no longer tests anything from `libtirpc`? We probably need to link in + call at least one symbol here.

**Review by iMichka — APPROVED** [review]

**Pull request merged**